plugins/easyimage/plugin.js

```diff
@@ -154,6 +183,9 @@
 					} else {
 						this.removeClass( editor.config.easyimage_sideClass );
 					}
+
+					editor.getCommand( 'easyimageFull' ).refresh( editor );
```

Comment:
I'll agree that this solution is not the most optimal. We'll need to change it in a followup task.